crates/bhyve-api/sys/src/vmm_data.rs

```diff
@@ -189,16 +190,23 @@ impl Default for vdi_rtc_v1 {
     }
 }
 
+// VDC_VMM_TIME v1 interface
+
+#[repr(C)]
+#[derive(Copy, Clone, Default, Serialize, Deserialize)]
```

Comment:
A few of the other devices are sending their vmm_data structs over the wire as the migration payload out of laziness (coming work will eliminate that).  Since that's not the case for the time data (since you have your own migrate type), it should be fine to omit `Serialize` and `Deserialize` here.

Reply:
done in cdbe3ef